python/tvm/relay/ir_builder.py

```diff
@@ -0,0 +1,299 @@
+"""IR builder for the Relay IR.
+
+Enables users to construct Relay programs with a Python API.
+"""
+from typing import Any
+import numpy as np
+import tvm
+from .type import FuncType, TensorType
+from .expr import Expr, Constant, Let, Var, Param, Function, If
+from .env import Environment
+
+
+def convert(arg: Any, ctxt=tvm.cpu(0)) -> tvm.nd.NDArray:
+    """Convert Python values into the appropriate types
+       for the Relay evaluator.
+    """
+    if isinstance(arg, int):
+        return tvm.nd.array(np.array(arg, dtype='int32'), ctxt)
+    elif isinstance(arg, float):
+        return tvm.nd.array(arg, ctxt)
+    elif isinstance(arg, bool):
+        return tvm.nd.array(np.array(arg, dtype='float32'), ctxt)
+    elif isinstance(arg, np.ndarray):
+        return tvm.nd.array(arg, ctxt)
+    elif isinstance(arg, tvm.ndarray.NDArray):
+        return arg
+    else:
+        # raise Exception(f"can't convert {type(arg)} to a Relay AST")
+        raise Exception(f"unsupported argument type {type(arg)}")
+
+
+def into_ast(arg: Any, ctxt=tvm.cpu(0)) -> Expr:
```

Comment:
better to name `to_ast` ?
add doc for this function

Reply:
👍